config.toml

```diff
@@ -4,11 +4,11 @@ title = "Fernglasz"
   [[menu.main]]
     identifier = "login"
     name = "Login"
-    url = "https://www.fernglasz.com/signin"
+    url = "https://fernglasz.web.app/signin"
```

Comment:
Make this relative URLs, so we do not have to change it every time

Reply:
Sure, will make them relative